nucliadb_models/src/nucliadb_models/graph/requests.py

```diff
@@ -161,8 +160,16 @@ class GraphSearchRequest(BaseGraphSearchRequest):
             "Filtering examples can be found here: https://docs.nuclia.dev/docs/rag/advanced/search-filters "
         ),
     )
-    security: Optional[RequestSecurity] = SearchParamDefaults.security.to_pydantic_field()
-    show_hidden: bool = SearchParamDefaults.show_hidden.to_pydantic_field()
+    security: Optional[RequestSecurity] = Field(
```

Comment:
Why this change?

Reply:
A problematic circular dependency between search and graph models. We could move SearchParamDefaults outside but still, feels kind of weird having a giant class with defaults for all endpoints